### PR TITLE
feat(fa): Bank information added to application

### PR DESCRIPTION
**Pull request by MargretFinnboga**
# Bank information form

https://app.asana.com/0/1201600280110882/1201624216643418

## What

- Added bank information and styles for them 

## Why

 - Optional information for application, however useful

## Screenshots / Gifs
Screenshot 2022-01-12 at 17 24 32

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/financial-aid/src/fields/BankInfoForm/BankInfoForm.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <>
+      <Text marginTop={2} marginBottom={[2, 2, 4]}>
+        {formatMessage(bankInfoForm.general.info)}
+      </Text>
+      <Box className={styles.bankInformationContainer}>
+        <Box className={styles.bankNumber} marginBottom={[2, 2, 4]}>
+          <InputController
+            id={`${id}.bankNumber`}
+            name={`${id}.bankNumber`}
+            defaultValue={`${answers}.bankNumber`}
```
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/fields/BankInfoForm/BankInfoForm.tsx:24]: Nor here

File: libs/application/templates/financial-aid/src/fields/BankInfoForm/BankInfoForm.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            label={formatMessage(bankInfoForm.inputsLabels.ledger)}
+            format="##"
+            backgroundColor="blue"
+          />
+        </Box>
+        <Box marginBottom={[2, 2, 4]} className={styles.accountNumber}>
+          <InputController
+            id={`${id}.accountNumber`}
+            name={`${id}.accountNumber`}
+            defaultValue={`${answers}.accountNumber`}
```
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/fields/BankInfoForm/BankInfoForm.tsx:45]: Nor here 👼

File: libs/application/templates/financial-aid/src/lib/dataSchema.ts [hunk 3]
```diff
@@ -72,6 +72,11 @@ export const dataSchema = z.object({
       params: error.validation.inputErrorMessage,
       path: ['custom'],
     }),
+  bankInfoForm: z.object({
+    bankNumber: z.string().optional(),
+    ledger: z.string().optional(),
+    accountNumber: z.string().optional(),
+  }),
```
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/lib/dataSchema.ts:79]: Have you tried the validation with every field empty? Does that work?

**Review by Rikkiabb — COMMENTED** [review]
One question :)

File: libs/application/templates/financial-aid/src/lib/dataSchema.ts (hunk 3 above)
Comment (MargretFinnboga) [reply libs/application/templates/financial-aid/src/lib/dataSchema.ts:79]: yess it works.. i think, at least in locally

**Pull request merged**